terraform/projects/infra-security-groups/main.tf

```diff
@@ -105,80 +105,22 @@ resource "aws_security_group" "monitoring_external_sg" {
   )}"
 }
 
-resource "aws_security_group" "alertmanager_external_sg" {
-  name        = "${var.stack_name}-alert-external-sg"
-  vpc_id      = "${data.terraform_remote_state.infra_networking.vpc_id}"
-  description = "Controls external access to the LBs"
-
-  tags = "${merge(
-    local.default_tags,
-    var.additional_tags,
-    map("Stackname", "${var.stack_name}"),
-    map("Name", "${var.stack_name}-monitoring-external-sg")
-  )}"
-}
-
-resource "aws_security_group_rule" "alertmanager_external_sg_ingress_any_http" {
-  type              = "ingress"
-  to_port           = 80
-  from_port         = 80
-  protocol          = "tcp"
-  security_group_id = "${aws_security_group.alertmanager_external_sg.id}"
-  cidr_blocks       = ["${var.cidr_admin_whitelist}"]
```

Comment:
Looking over this rule with @DavidJeche, we can't understand what it was supposed to be doing. It allows http access directly from our office IPs to the internal load balancer.  Why did we ever want this?

If I try to visit the internal ALB (by copypasting it's default DNS record into a browser), it just times out, suggesting that even with this rule, access isn't permitted.  Did it ever work?  Since we always have accessed alertmanager via the nginx auth proxy (not directly), I'm not sure we've ever seen this rule in action.